extensions/jdbc/src/main/java/io/deephaven/jdbc/JdbcTypeMapper.java

```diff
@@ -669,37 +638,9 @@ public void bindFromChunk(Chunk<Values> srcChunk, int srcOffset, PreparedStateme
         }
     }
 
-    public static class ByteDataTypeMapping extends DataTypeMapping<Byte> {
-        ByteDataTypeMapping() {
-            super(Types.TINYINT, Byte.class);
-        }
-
-        @Override
-        public void bindToChunk(
-                WritableChunk<Values> destChunk, int destOffset,
-                ResultSet resultSet, int columnIndex, Context context) throws SQLException {
-            final WritableByteChunk<Values> byteChunk = destChunk.asWritableByteChunk();
-            final byte value = resultSet.getByte(columnIndex);
-            byteChunk.set(destOffset, resultSet.wasNull() ? QueryConstants.NULL_BYTE : value);
-        }
-
-        @Override
-        public void bindFromChunk(
-                Chunk<Values> srcChunk, int srcOffset,
-                PreparedStatement stmt, int parameterIndex, Context context) throws SQLException {
-            final ByteChunk<Values> byteChunk = srcChunk.asByteChunk();
-            final byte value = byteChunk.get(srcOffset);
-            if (value == QueryConstants.NULL_BYTE) {
-                stmt.setNull(parameterIndex, sqlType);
-            } else {
-                stmt.setByte(parameterIndex, value);
-            }
-        }
-    }
-
     public static class BooleanDataTypeMapping extends DataTypeMapping<Boolean> {
         BooleanDataTypeMapping(int sqlType) {
-            super(sqlType, Boolean.class);
+            super(sqlType, boolean.class);
```

Comment:
Not so sure about this change. We always used boxed `Boolean` in Deephaven columns.